### PR TITLE
Offline tile restoration

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: src/offline/Restorer.js [hunk 1]
```diff
@@ -20,15 +20,22 @@ class Restorer {
...
+    return this.configuration_.getItem('offline_content').then(offlineContent => this.doRestore_(map, offlineContent));
   }
 
   /**
+   * @param {ol.Map} map
    * @param {ngeox.OfflinePersistentContent} offlineContent
    * @return {ol.Extent}
    */
-  doRestore_(offlineContent) {
+  doRestore_(map, offlineContent) {
```
Comment (fgravin) [inline src/offline/Restorer.js]: this is were we could override what we do with the layers ...

**Comment by gberaudo** [comment]
@fgravin, @ger-benjamin, I implemented storing/restoration of layer metadata and background layer.
It is working nicely in debug mode; not yet in compiled mode. I will see what is the reason for that.

File: src/offline/Restorer.js (hunk 1 above)
Comment (ger-benjamin) [reply src/offline/Restorer.js]: add a `@private` or remove the final underscore please

Comment (gberaudo) [reply src/offline/Restorer.js]: OK, done. Anything else to do?

File: src/offline/Restorer.js [hunk 2]
```diff
@@ -20,20 +30,33 @@ class Restorer {
    * @return {Promise<ol.Extent>}
    */
   restore(map) {
-    console.log('should restore something');
-    return this.configuration_.getItem('offline_content').then(offlineContent => this.doRestore_(offlineContent));
+    return this.configuration_.getItem('offline_content').then(offlineContent => this.doRestore(map, offlineContent));
   }
 
   /**
+   * @protected
```
Comment (ger-benjamin) [inline src/offline/Restorer.js]: equal to nothing I guess

**Comment by fgravin** [comment]
looks good so far

**Comment by gberaudo** [comment]
@ger-benjamin, thanks for the review. I am willing to merge as soon as it is green so that it can be tested this end of afternoon and delivered tomorrow.

**Pull request merged**